app/javascript/controllers/addMember_controller.js

```diff
@@ -0,0 +1,55 @@
+import { Controller } from 'stimulus';
+import 'bootstrap-tagsinput/dist/bootstrap-tagsinput.js';
+import * as $ from 'jquery/dist/jquery';
+import 'bootstrap/dist/js/bootstrap';
+
+export default class extends Controller {
+    connect() {
```

Comment:
Can you consider refactoring using  data-action https://stimulus.hotwired.dev/handbook/building-something-real

Reply:
Sure Sir.